### DISCUSSION TITLE
Add modifier keys to print statements

### DISCUSSION THREAD

**Discussion by jonbell-lot23** [RecordReplay/devtools#7134] [Ideas]
A few things to frame this up:

1. Modifier keys are great, but it's important they not be the only way to do something. But..
2. If the feature is pro enough, you can get away with it being the only way to do something
3. Modifier keys provide a great way to offer power features without cluttering the interface

As part of our ["jump straight to a hit point" discussion](https://github.com/replayio/devtools/discussions/7133) , @markerikson and @jonbell-lot23 discussed adding keyboard modifiers as a way to jump directly to hits in the print statement panel.

Some ideas:
* A modifier key to jump 10 hits at a time
* A modifier key to jump to the start or end
* A modifier key to click and drag as a way to "scrub" through numbers, Bret Victor style (cc @souporserious )

**Comment by jasonLaster** [comment]
> "scrub" through numbers

This feels like something the badge while in edit mode could do...
